Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib64/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib64/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/utils/autoreload.py", line 54, in wrapper
    fn(*args, **kwargs)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/core/management/commands/runserver.py", line 117, in inner_run
    self.check(display_num_errors=True)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/core/management/base.py", line 390, in check
    include_deployment_checks=include_deployment_checks,
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/core/management/base.py", line 377, in _run_checks
    return checks.run_checks(**kwargs)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/core/checks/registry.py", line 72, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/core/checks/urls.py", line 40, in check_url_namespaces_unique
    all_namespaces = _load_all_namespaces(resolver)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/core/checks/urls.py", line 57, in _load_all_namespaces
    url_patterns = getattr(resolver, 'url_patterns', [])
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/urls/resolvers.py", line 579, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/urls/resolvers.py", line 572, in urlconf_module
    return import_module(self.urlconf_name)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib64/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/wercules/medreport/mrProject/urls.py", line 20, in <module>
    path('mr/', include('mrApp.urls')),
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib64/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/wercules/medreport/mrApp/urls.py", line 5, in <module>
    from . import views
  File "/home/wercules/medreport/mrApp/views.py", line 9, in <module>
    from .forms import PacienteForm, ProcurarPacienteForm, AtendimentoForm
  File "/home/wercules/medreport/mrApp/forms.py", line 3, in <module>
    from mrApp.forms import Paciente, Atendimento, Receituario
ImportError: cannot import name 'Paciente'

/home/wercules/medreport/mrApp/forms.py changed, reloading.
Performing system checks...

Performing system checks...

Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
[20/Jun/2019 14:19:43] "GET /mr/ HTTP/1.1" 200 2439
Not Found: /resources/demos/style.css
[20/Jun/2019 14:19:43] "GET /resources/demos/style.css HTTP/1.1" 404 2241
[20/Jun/2019 14:19:43] "GET /static/js/main.js HTTP/1.1" 304 0
[20/Jun/2019 14:19:43] "GET /static/css/style.css HTTP/1.1" 304 0
[20/Jun/2019 14:19:49] "POST /mr/procurar_paciente HTTP/1.1" 200 2418
Not Found: /resources/demos/style.css
[20/Jun/2019 14:19:49] "GET /resources/demos/style.css HTTP/1.1" 404 2241
[20/Jun/2019 14:19:51] "GET /mr/cadastrar_atendimento?paciente_id=13 HTTP/1.1" 200 2554
Not Found: /resources/demos/style.css
[20/Jun/2019 14:19:51] "GET /resources/demos/style.css HTTP/1.1" 404 2241
/home/wercules/medreport/mrApp/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
June 20, 2019 - 14:19:28
Django version 2.2.1, using settings 'mrProject.settings'
Starting development server at http://cardiobemestar.com.br:8000/
Quit the server with CONTROL-C.
Watching for file changes with StatReloader
/home/wercules/medreport/mrApp/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
June 20, 2019 - 14:22:13
Django version 2.2.1, using settings 'mrProject.settings'
Starting development server at http://cardiobemestar.com.br:8000/
Quit the server with CONTROL-C.
Watching for file changes with StatReloader
Internal Server Error: /mr/cadastrar_atendimento
Traceback (most recent call last):
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/wercules/.local/share/virtualenvs/medreport-3oUkA-aL/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/wercules/medreport/mrApp/views.py", line 154, in cadastrar_atendimento
    return render(request, 'cadastrar_atendimento.html', {'form_anamnese': form_anamnese, 'context':context, 'form_receituario':form_receituario})
NameError: name 'form_anamnese' is not defined
[20/Jun/2019 14:22:48] "GET /mr/cadastrar_atendimento?paciente_id=13 HTTP/1.1" 500 68596
/home/wercules/medreport/mrApp/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
June 20, 2019 - 14:22:17
Django version 2.2.1, using settings 'mrProject.settings'
Starting development server at http://cardiobemestar.com.br:8000/
Quit the server with CONTROL-C.
Watching for file changes with StatReloader
[20/Jun/2019 14:23:51] "GET /mr/cadastrar_atendimento?paciente_id=13 HTTP/1.1" 200 3715
Not Found: /resources/demos/style.css
[20/Jun/2019 14:23:52] "GET /resources/demos/style.css HTTP/1.1" 404 2241
/home/wercules/medreport/mrApp/forms.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
June 20, 2019 - 14:23:42
Django version 2.2.1, using settings 'mrProject.settings'
Starting development server at http://cardiobemestar.com.br:8000/
Quit the server with CONTROL-C.
Watching for file changes with StatReloader
[20/Jun/2019 14:24:56] "GET /mr/cadastrar_atendimento?paciente_id=13 HTTP/1.1" 200 3437
[20/Jun/2019 14:24:56] "GET /static/css/style.css HTTP/1.1" 304 0
Not Found: /resources/demos/style.css
[20/Jun/2019 14:24:56] "GET /resources/demos/style.css HTTP/1.1" 404 2241
[20/Jun/2019 14:24:56] "GET /static/js/main.js HTTP/1.1" 304 0
[20/Jun/2019 14:37:55] "GET /mr/cadastrar_atendimento?paciente_id=13 HTTP/1.1" 200 3439
Not Found: /resources/demos/style.css
[20/Jun/2019 14:37:55] "GET /resources/demos/style.css HTTP/1.1" 404 2241
